engine/deck.py

```diff
@@ -30,3 +40,24 @@ def draw_card(self):
         if len(self.card_list) > 0:
             return self.card_list.pop()
         return False
+
+
+class WildDeck(Deck):
```

Comment:
Wouldn't it make more sense to keep has_wild as a class attribute for Deck (which we'd pass to board for AIs) while instead having colors manage an attribute "is_wild"?  Subclassing Deck for wild is weird IMO because both are a "deck", one just contains a wild suit.  Especially if you're keeping color initialization as passing classes, I would just make a parameter for one of the colors you pass.